Yona/Yona/nl.lproj/Localizable.strings

```diff
@@ -6,6 +6,11 @@
  Copyright © 2016 Yona. All rights reserved.
  */
 
+//Switch UI
+"environment-alert-title" = "Voer hieronder de URL van de omgeving in:";
+"environment-success" = "U bevindt zich nu in:";
+"environment-failure" = "Omgevingsschakelaar Fout. Teruggekeerd naar:";
```

Comment:
Make this: "Wisselen van server mislukt. Teruggekeerd naar: "

Reply:
Updated